src/pages/MovieDetail/components/CommentList.js

```diff
@@ -0,0 +1,198 @@
+import { React, useState } from 'react';
+// import axios from 'axios';
+import styled from 'styled-components';
+import Modal from './Modal';
+import { SiMega } from 'react-icons/si';
+import { TbPencil } from 'react-icons/tb';
+
+const CommentList = ({ movieInfo }) => {
+  const [commentModal, setCommentModal] = useState(false);
+  const [clickStar, setClickStar] = useState([
+    false,
+    false,
+    false,
+    false,
+    false,
+  ]);
+  const [image, setImage] = useState({
+    image_file: '',
+    preview_URL: 'images/default.png',
+  });
+  const [loaded, setLoaded] = useState(false);
+
+  const openModal = () => {
+    setCommentModal(true);
+  };
+
+  const closeModal = () => {
+    setCommentModal(false);
+  };
+
+  const starRating = index => {
+    let clickStates = [...clickStar];
+    for (let i = 0; i < 5; i++) {
+      clickStates[i] = i <= index ? true : false;
```

Comment:
조건 자체가 boolean이기 때문에 조건만 사용하시면 됩니다.
```suggestion
      clickStates[i] = i <= index;
```